comp/metadata/inventoryhost/inventoryhostimpl/inventoryhost.go

```diff
@@ -83,6 +83,7 @@ type hostMetadata struct {
 	IPAddress   string `json:"ip_address"`
 	IPv6Address string `json:"ipv6_address"`
 	MacAddress  string `json:"mac_address"`
+	Interfaces  string `json:"interfaces"`
```

Comment:
If this is a new element, will it be taken into account in the backend ?

Reply:
The plan is to use it in FA, so yes